### ISSUE TITLE
Support already made playlists

### ISSUE THREAD

**Issue by lubien** [s-kris/rey#2]
I often use my own playlists on YouTube such as: https://www.youtube.com/watch?v=7o5jGiLeVB8&list=PLRsrBMpg-WP9e7g-iKsPlv91s_X4xCOBL&index=33

But your player currently doesn't support playlists from youtube.

![image](https://user-images.githubusercontent.com/9121359/45554244-50d42980-b80c-11e8-8b6a-407da455a69e.png)

That's a huge drawback for me since it's not ideal to rebuild my somewhat large playlists once again.

**Comment by s-kris** [comment]
I agree with the playlists problem. Until yesterday there were like 3 daily active users and 1 of them being me :D. Will try to implement that feature in future.

**Comment by lubien** [comment]
Thanks! Take your time, I'll be looking forward.
